src/lib389/lib389/cli_conf/replication.py

```diff
@@ -318,12 +318,10 @@ def get_repl_status(inst, basedn, log, args):
     replicas = Replicas(inst)
     replica = replicas.get(args.suffix)
     if args.bind_passwd_file is not None:
-        passwd = get_passwd_from_file(args.bind_passwd_file)
-    elif args.bind_passwd_prompt:
-        passwd = _get_arg(None, msg=f"Enter password for ({args.bind_dn})", hidden=True, confirm=True)
-    else:
-        passwd = args.bind_passwd
-    status = replica.status(binddn=args.bind_dn, bindpw=passwd)
+        args.bind_passwd = get_passwd_from_file(args.bind_passwd_file)
+    if args.bind_passwd_prompt or args.bind_dn is None or args.bind_passwd is None:
```

Comment:
Is bind_dn set when using GSSAPI ?

Reply:
The answer is not always but `dsconf instanceName replication status`  does not provide any parameter to specify the auth method, so there is no way to use GSSAPI anyway ...